test/parallel/test-tick-processor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var fs = require('fs');
+var assert = require('assert');
+var path = require('path');
+var cp = require('child_process');
+var common = require('../common');
+
+common.refreshTmpDir();
+
+var prevDir = process.cwd();
+process.chdir(common.tmpDir);
+var f = cp.execFileSync(process.execPath, ['-prof', '-pe',
+    'function fib(n) { return n < 2 ? n : fib(n-1) + fib(n-2); }; fib(40);']);
+fs.readdir(common.tmpDir, function(err, files) {
+  if (err) {
+    assert.fail(err);
+  }
+  var matches = [];
+  for (var i in files) {
+    if (files[i].length > 8 && files[i].substr(0, 8) === 'isolate-') {
+      matches.push(files[i]);
+    }
+  }
+  if (matches.length != 1) {
+    assert.fail('There should be a single log file.');
+  }
+  var log = matches[0];
+  var processor = path.join(common.testDir, '..', 'tools', 'v8-prof',
+      getScriptName());
+  var child;
+  if (process.platform === 'win32') {
+    child = cp.spawn('cmd.exe', ['/c', processor, log]);
+  } else {
+    child = cp.spawn(processor, [log]);
+  }
+  var out = '';
+  child.stdout.on('data', function(buf) {
+    out += buf;
+  });
+  child.on('error', function(err) {
+    assert.fail(err);
+  });
+  child.on('close', function(code) {
+    assert(out.match(/LazyCompile.*fib/));
+    process.chdir(prevDir);
+  });
```

Comment:
Can I suggest you use `cp.execSync()` here for ease of use?  That should also let you remove the win32 code path.

Aside, it doesn't hurt but I don't think you have to `process.chdir()` back to the original working directory.